### PR TITLE
MC-14316: Updated api docs for custom purge

### PR DESCRIPTION
**Pull request by RGitB**
#### Changes made

- Updated docs for custom cache purge

### CODE REVIEW

File: source/includes/stackpath/_sites.md [hunk 1]
```diff
@@ -474,8 +475,9 @@ Purge custom CDN cached content of a site in a given [environment](#administrati
 
 Required | &nbsp;
 ------- | -----------
+`purgeType`<br/>*String* | The type of cache purge. Can be `URL` or `PATH`.
 `items`<br/>*Array[Object]* | The items to purge from the CDN.
-`items.url`<br/>*String* | The URL at which to delete content.
+`items.url`<br/>*String* | The URL or Path at which to delete content.
```
Comment (fyrnaga) [inline source/includes/stackpath/_sites.md]:
```suggestion
`items.url`<br/>*String* | The URL or path at which to delete content.
```

**Review by fyrnaga — APPROVED** [review]
small correction but look ok

File: source/includes/stackpath/_sites.md [hunk 2]
```diff
@@ -442,6 +442,7 @@ curl -X POST \
 
 ```json
 {
+  "purgeType":"PATH",
```
Comment (GuillaumeVialle) [inline source/includes/stackpath/_sites.md]: I don't think the purge type should be needed. The API supports other types than URL and PATH, and you can have a mix of both.

File: source/includes/stackpath/_sites.md (hunk 1 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: 1994020

File: source/includes/stackpath/_sites.md (hunk 2 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: Changed it to optional field. Default type is URL now
74776cc

**Review by GuillaumeVialle — APPROVED** [review]
Thanks for the change

**Pull request merged**